msteams-platform/Teams-AI-library-tutorial.yml

```diff
@@ -24,6 +24,7 @@ items:
     * How to interact with your LLMs and data.
     * The directory structure of your custom engine agent.
 
+    Once you have a working custom engine agent on Teams, signal the intent to participate in Microsoft 365 Copilot by explicitly declaring `customEngineAgents` in the app manifest.
```

Comment:
As this is not released yet, shall we remove it?

**explicitly declaring `customEngineAgents` in the app manifest.**

Reply:
Removed this sentence as suggested.

Reply:
@v-sdhakshina - If customEngineAgents  is supported in developer preview. We no need to remove it.